src/resources/filters/quarto-post/latex.lua

```diff
@@ -376,6 +376,13 @@ function render_latex()
       end
       
       return pandoc.Div(calloutContents)
-    end    
+    end,
+    Note = function(n)
+      if marginReferences() then
```

Comment:
I think there's a couple of different ways that notes can end up in the margin. marginReferences() is one of them, but I think there's a way to annotate a div, in which case we need to do something slightly different, by catching the wrapping div first.

Reply:
Thanks - I'll look into that then...

Reply:
So after looking at others way to do margin content, I do think only `sidenotes` is used for `footnotes` in margin. 

So probably the current solution is enough to cover the issue; 


overview of other ways below

## [Margin Content](https://quarto.org/docs/authoring/article-layout.html#margin-content)

Using `.column-margin` div 
````markdown
::: {.column-margin}

This is a text of several paragraphs in the margin.

Does it works ? 
:::

````

does not have issues because in LaTeX, `\marginnote` command is used, not `\sidenote`

````latex
\marginnote{\begin{footnotesize}

This is a text of several paragraphs in the margin.

Does it works ?

\end{footnotesize}}
````

## [Margin Figures](https://quarto.org/docs/authoring/article-layout.html#margin-figures)

For figures in margin, `\begin{marginfigure}` is used. 

## [Margin Tables](https://quarto.org/docs/authoring/article-layout.html#margin-tables)

`\begin{margintable}` is used, which works ok. 


## [Caption in Margin](https://quarto.org/docs/authoring/article-layout.html#margin-captions)

`\sidecaption` is used.